src/modules/collection/reducer.spec.ts

```diff
@@ -51,6 +52,44 @@ describe('when FETCH_TRANSACTION_SUCCESS', () => {
   })
 })
 
+describe('when FETCH_COLLECTIONS_REQUEST', () => {
+  const anExistingCollectionId = 'id'
+  let mockedFetchCollectionParams: FetchCollectionsParams
+  let initialState: CollectionState
+  beforeEach(() => {
+    mockedFetchCollectionParams = {
+      assignee: 'anAssignee',
+      isPublished: false
+    } as FetchCollectionsParams
+    initialState = {
+      data: {
+        [anExistingCollectionId]: {
+          id: 'anExistingCollectionId'
+        }
+      },
+      lastFetchParams: mockedFetchCollectionParams,
+      loading: []
+    } as any
+  })
+
+  describe('and it sends the flag to re-use existing results and same parameters', () => {
+    it('should not set the loading state', () => {
+      const state = reducer(initialState, fetchCollectionsRequest(undefined, mockedFetchCollectionParams, true))
+      expect(reducer(initialState, fetchCollectionsRequest(undefined, mockedFetchCollectionParams, true))).toEqual(state)
+    })
```

Comment:
If I'm not wrong, you are missing the case where the actions has the re-use flag but the parameters don't match, would you mind adding a test for this case?

Reply:
added!